docs/src/man/getting_started.md

```diff
@@ -69,3 +59,39 @@ Pkg.checkout("LazySets")
 ```
 That will check out the latest version in the `master` branch, and precompile it
 the next time you enter a session and execute `using LazySets`.
+
+## Optional dependencies
+
+An optional dependency is a package that is not required to compile and use `LazySets.jl`,
+although some extra functionality is available provided that you load that package. Internally, optional
+dependencies in Julia are handled with the package [Requires.jl](https://github.com/JuliaPackaging/Requires.jl).
+
+For example, if you want to work with sets defined symbolically as in `var = @variables x[1:10]; Hyperplane(x[1] + x[2] == 1/2, var)` to define the hyperplane $H : \{ x \in \mathbb{R}^n : x_1 + x_2 == 1/2\}$, install [ModelingToolkit.jl](https://github.com/SciML/ModelingToolkit.jl) so that such constructor (and other symbolic features) are automatically available once doing `using ModelingToolkit` in your session.
+Another optional dependency is [`Expokit.jl`](https://github.com/acroy/Expokit.jl), a package that provides
+lazy matrix exponentiation routines for linear maps of `LazySets` types.
+
+The following table summarizes each optional dependency and the features available together with `LazySets.jl`:
+
+|Dependency|Features|
+|----------|-------|
+|`CDDLib` |Polyhedral computations backend|
+|`Distributions` |Random sampling|
+|`Documenter` |Building the projects' documentation|
+|`Expokit` |Computing the action of a matrix exponential over a set|
+|`GR` |Plotting backend|
+|`IntervalConstraintProgramming` |Conservative polyhedral approximation of a region defined implicitly via nonlinear constraints|
+|`IntervalMatrices` |Set operations that involve matrices whose coefficients are intervals|
+|`Makie` |Plotting library, mainly for 3D and interactive plots|
+|`MiniQhull` |Voronoi-Delaunay triangularization of `LazySets` types|
+|`ModelingToolkit` |Create sets using symbolic expressions|
+|`Optim` |Numerical optimization package|
+|`Plots` |Plotting library, mainly 2D plots|
+|`Polyhedra` |Concrete polyhedra library|
+|`StaticArrays` |Statically defined arrays, mainly for low dimensions|
+|`TaylorModels` |Taylor expansion of functions with rigorous interval remainder|
+
+Use the following command to install *all* optional dependencies:
+
+```julia
+julia> ] add CDDLib Distributions Documenter Expokit GR IntervalConstraintProgramming IntervalMatrices Makie MiniQhull ModelingToolkit Optim Plots Polyhedra StaticArrays TaylorModels
```

Comment:
the reason to have this list was that it was been a pain the couple of times that i began to develop LazySets in a fresh machine: you have to install all the optional dependencies by hand, so having this command is useful.

i agree to remove `GR` since it is installed by default with Plots if no other backend is used (i think). about Documenter, i can remove it from this list and mention it in a separate line.

Reply:
fixed